Gemfile.lock

```diff
@@ -221,6 +221,8 @@ GEM
       jekyll-feed (~> 0.9)
       jekyll-seo-tag (~> 2.1)
     minitest (5.16.1)
+    nokogiri (1.13.9-arm64-darwin)
```

Comment:
As in the other Pull Request, could you revent your changes ou this file? Thanks

Reply:
will do

Reply:
@bebatut I believe this one is fixed 🙂